signer/src/message.rs

```diff
@@ -208,6 +214,8 @@ impl SweepTransactionInfo {
                 .signers_script_pubkey(),
             amount: unsigned.output_amounts(),
             fee: unsigned.tx_fee,
+            // the vsize should never be larger than u32::MAX
```

Comment:
Oh because the max block vsize is currently `u32::MAX` bytes, right?

Reply:
That and that it would be completely insane if we had a transaction of this size; max bitcoin block size is ~4mb.

Reply:
This is changed in [a399760](https://github.com/stacks-network/sbtc/pull/810/commits/a39976075a79f2c064b214d368485b787a1139e0) from the above comment